ChangeLog.md

```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 3.1.75 (in development)
 -----------------------
+- `FS.basename()` no longer calls `PATH.normalize()`, so that
+  `FS.basename("a/.")` returns `"."` instead of `"a"` and `FS.basename("a/b/..")`
+  returns `".."` instead of `"a"`. (#23180)
```

Comment:
How about adding "This is in line with the behaviour of both node and coreutils, and is already the case when using NODEFS/NODERAWFS".

Reply:
`s/FS.basename/PATH.basename/`

Reply:
I think it's only already the case in NODERAWFS and is the cause of bugs in NODEFS.

Reply:
Did you see s/FS.basename/PATH.basename/ ?

Reply:
Oops, updated that now.